resources/lrsql/doc/openapi.yaml

```diff
@@ -0,0 +1,790 @@
+          member:
+            type: array
+            items: {$ref: '#/components/schemas/Agent'}
+        required: [objectType, member]
+      - allOf:
+        - properties:
+            objectType: {type: string, pattern: Group}
+            name: {type: string}
+            member:
+              type: array
+              items: {$ref: '#/components/schemas/Agent'}
+          required: [objectType]
+        - {$ref: '#/components/schemas/IFI'}
+    IRI: {type: string, format: iri}
+    statementId: {type: string}
+    ScopedKeyPair:
+      allOf:
+      - {$ref: '#/components/schemas/KeyPair'}
+      - {$ref: '#/components/schemas/Scopes'}
+    IRL: {type: string}
+    KeyPair:
+      type: object
+      properties:
+        api-key: {type: string}
+        secret-key: {type: string}
+      required: [api-key, secret-key]
+    StatementResult:
+      type: object
+      required: [statements]
+      properties:
+        statements:
+          type: array
+          items: {$ref: '#/components/schemas/Statement'}
+        more: {$ref: '#/components/schemas/IRL'}
+    IFI:
+      oneOf:
+      - type: object
+        properties:
+          mbox: {$ref: '#/components/schemas/MailToIRI'}
+        required: [mbox]
+      - type: object
+        properties:
+          mbox_sha1sum: {type: string}
+        required: [mbox_sha1sum]
+      - type: object
+        properties:
+          openid: {$ref: '#/components/schemas/URI'}
+        required: [openid]
+      - type: object
+        properties:
+          account: {$ref: '#/components/schemas/Account'}
+        required: [account]
+    Scopes:
+      type: object
+      properties:
+        scopes:
+          type: array
+          items: {type: string}
+      required: [scopes]
+    URI: {type: string, format: uri}
+    Actor:
+      oneOf:
+      - {$ref: '#/components/schemas/Group'}
+      - {$ref: '#/components/schemas/Agent'}
+    Account:
+      type: object
+      properties:
+        homePage: {$ref: '#/components/schemas/IRL'}
+        name: {type: string}
+      required: [homePage, name]
+    Timestamp: {type: string, format: date-time}
+    Error:
+      type: object
+      properties:
+        error: {type: string}
+      required: [error]
+    UUID: {type: string, format: uuid}
+    Person:
+      type: object
+      properties:
+        objectType: {type: string, pattern: Person}
+        name:
+          type: array
+          items: {type: string}
+        mbox:
+          type: array
+          items: {$ref: '#/components/schemas/MailToIRI'}
+        mbox_sha1sum:
+          type: array
+          items: {type: string}
+        openid*:
+          type: array
+          items: {$ref: '#/components/schemas/URI'}
+        account*:
+          type: array
+          items: {$ref: '#/components/schemas/Account'}
+      required: [objectType]
+    Statement: {type: object, description: 'https://github.com/adlnet/xAPI-Spec/blob/master/xAPI-Data.md#20-statements'}
```

Comment:
this might be a good place for a `$comment` about the fact that the statement spec is intentionally omitted